### PR TITLE
[update] SQL function API Reference

### LINKED ISSUE #837: [Docs RFC] Update Promscale deletion docs to mention delete_series_from_metric()
# Describe change in content, appearance, or functionality

Docs mention deletion of series via the HTTP API but not the SQL way of running `_prom_catalog.delete_series_from_metric(name text, series_ids bigint[])`

[Relevant Slack message](https://timescaledb.slack.com/archives/C011FC0PPC5/p1645168130769889)

# Subject matter expert (SME)

@antekresic 

# Deadline

[When does this need to be addressed]

# Any further info

[Anything else you want to add, or further links]

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

# Describe change in content, appearance, or functionality

Docs mention deletion of series via the HTTP API but not the SQL way of running `_prom_catalog.delete_series_from_metric(name text, series_ids bigint[])`

[Relevant Slack message](https://timescaledb.slack.com/archives/C011FC0PPC5/p1645168130769889)

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/837

### CODE REVIEW

**Comment by VineethReddy02** [comment]
@Rajakavitha1 as you have sorted it alphabetically it's hard to understand the exact difference. So for delete series all you have added this record right?

```
delete_series_from_metric | metric_name text, series_ids| boolean | deletes the series from the metric
```

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 as you have sorted it alphabetically it's hard to understand the exact difference. So for delete series all you have added this record right?
> 
> ```
> delete_series_from_metric | metric_name text, series_ids| boolean | deletes the series from the metric
> ```

Yes, that is right. I also added the functions that were missing in the docs from [promscale sql api ](https://github.com/timescale/promscale/blob/master/docs/sql_api.md)

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

File: promscale/sql-api.md [hunk 1]
```diff
@@ -19,21 +19,35 @@ ORDER BY 1, 2, 4;
...
- set_metric_chunk_interval     | metric_name text, chunk_interval interval                | boolean          | set_metric_chunk_interval set a chunk interval for a specific metric (this overrides the default).
- set_metric_retention_period   | metric_name text, new_retention_period interval          | boolean          | set_metric_retention_period set a retention period for a specific metric (this overrides the default).
- val                           | label_id integer                                         | text             | val returns the label value from a label id.
- unregister_metric_view        | schema_name text, view_name text, if_not_exists boolean  | boolean          | Unregister metric view with Promscale. Schema name and view name should be set to the metric view already registered in Promscale. 
\ No newline at end of file
+ add_prom_node | node_name text, attach_to_existing_metrics boolean DEFAULT true |  |
+ config_maintenance_jobs | number_jobs integer, new_schedule_interval interval, new_config jsonb DEFAULT NULL::jsonb| boolean | Configure the number of maintence jobs run by the job scheduler, as well as their scheduled interval.
```
Comment (Loquacity) [inline promscale/sql-api.md:23]:
```suggestion
 add_prom_node|node_name text, attach_to_existing_metrics boolean DEFAULT true||
config_maintenance_jobs|number_jobs integer, new_schedule_interval interval, new_config jsonb DEFAULT NULL::jsonb|boolean|Configure the number of maintenance jobs run by the job scheduler, as well as their scheduled interval.
```

Comment (Loquacity) [reply promscale/sql-api.md:23]: unspace entire table 👇

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: promscale/sql-api.md [hunk 2]
```diff
@@ -17,37 +17,37 @@ WHERE n.nspname OPERATOR(pg_catalog.~) '^(prom)$' COLLATE pg_catalog.default
 ORDER BY 1, 2, 4;
 -->
 
- Name | Arguments | Return type | Description
- --- | --- | --- | ---
+ Name|Arguments|Return type|Description
+  --- | --- | --- | ---
```
Comment (Loquacity) [inline promscale/sql-api.md:21]:
```suggestion
|-|-|-|-|
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: promscale/sql-api.md (hunk 2 above)
Comment (Loquacity) [reply promscale/sql-api.md:21]: @Rajakavitha1 Check that this table builds as expected - you probably need pipes at the beginning and end of the lines as well.

Comment (Rajakavitha1) [reply promscale/sql-api.md]: Hi @Loquacity ! I just checked the local build and the table appears fine.
Screenshot 2022-04-12 at 12 01 07 PM
Screenshot 2022-04-12 at 12 01 15 PM

**Pull request closed without merging**